app/tests/test_browse.py

```diff
@@ -513,18 +513,18 @@ def test_browse_consignment_with_missing_file(
             "Closure start date",
             "Closure period",
         ],
-        ["None", "test_file12.txt", "None", "None", "None"],
+        ["None", "test_file11.txt", "closed", "-", "-"],
+        ["None", "test_file12.txt", "closed", "-", "-"],
     ]
 
-    assert [
-        header.text.replace("\n", " ").strip(" ") for header in headers
-    ] == expected_results_table[0]
+    headers = [header.text.strip() for header in table.find_all("th")]
+    assert headers == expected_results_table[0]
 
-    for index, row in enumerate(rows):
-        values = row.find_all("dd")
-        assert [
-            value.text.replace("\n", " ").strip(" ") for value in values
-        ] == expected_results_table[index + 1]
+    for index, row in enumerate(rows[1:]):
+        values = [value.text.strip() for value in row.find_all("td")]
+        expected_values = expected_results_table[index + 1]
+
+        assert values == expected_values
```

Comment:
why did these asserts change structure, but not the one in `test_browse_consignment_filter_display_multiple_pages` for example

Reply:
There's now a <strong> which contains the data. It was just easier for me to understand the assertion. Assertions should be the same

Reply:
Revert the format of this test